app/views/display/one.html.slim

```diff
@@ -4,19 +4,17 @@
   p= 'Get Transcript allows you to access your personal tax information and proof of income information online.'
 .verbiage
   p
-    | Before you can access your transcript, you need to verify your identity.
-    | Secure Accounts is the new way to prove your identity online. This process will take about 10 minutes.
+    | Before you can access your transcript online, you need to verify your identity.
+    | This process will take about 10 minutes.
 .directives
-  p
-    'This is my first time here.
+  h3= 'This is my first time here.
```

Comment:
in this case you don't need the leading ' that's only for when you start the content on the next line (and in most cases, I think, we ought to use the | character. 

In most cases I think you don't need the = either, although there are cases when you do. I'll try to chase down the distinction.